packages/orbit-components/src/Layout/LayoutColumn/index.jsx

```diff
@@ -12,6 +12,11 @@ import type { Props } from ".";
 
 const StyledColumn = styled.div`
   ${({ hideOn }) => !!hideOn && getViewportHideStyles(hideOn)};
+  ${({ spanEntireRow }) =>
```

Comment:
could you rewrite to use only single arrow function with `css` from SC? We even have a eslint rule for that in a plugin :)